server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CounterResource.java

```diff
@@ -142,7 +124,6 @@ private static OperationFailedException counterManagerNotFound() {
         public static final CounterRemoveCommand INSTANCE = new CounterRemoveCommand();
 
         private CounterRemoveCommand() {
```

Comment:
There is no need for this constructor, or any of the constructors in the classes that extend `BaseCounterManagerCommand`